configs/native.yaml

```diff
@@ -1,10 +1,10 @@
 plugin_groups:
   - !include "rt_slam_plugins.yaml"
   - plugin_group:
-      - path: ground_truth_slam/
+      #- path: ground_truth_slam/
```

Comment:
Make sure to restore the config files to their previous state before merging this PR.